Tests/SentryTests/Helper/SentryAppStateManagerTests.swift

```diff
@@ -1,9 +1,9 @@
+import SentryTestUtils
 import XCTest
 
 #if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
 class SentryAppStateManagerTests: XCTestCase {
     private static let dsnAsString = TestConstants.dsnAsString(username: "SentryOutOfMemoryTrackerTests")
-    private static let dsn = TestConstants.dsn(username: "SentryOutOfMemoryTrackerTests")
```

Comment:
There were a bunch of these that weren't actually being used. I figured it was a copypasta.